### PR TITLE
Updated Contributing.md with workflow documentation.

### PR DESCRIPTION
**Pull request by ashu54-aseem**
## Fixes
Fixes issue #304 

## Description
Adds a simple workflow diagram to the contributing doc which would act as a shorthand guide for any new contributor to get a rough idea of how the app works.

## Checklist
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ project documentation. If you cannot find the documentation you're
 looking for, please file a GitHub issue with details of what
 you'd like to see documented.
 
+## Workflow of the App
+
+![Workflow Diagram](docs/img/workflow.jpeg)
```
Comment (Cronus1007) [inline CONTRIBUTING.md:24]: @ashu54-aseem @obulat @zackkrida I here will prefer a markdown rather than just an image. What's your opinion?

Comment (ashu54-aseem) [reply CONTRIBUTING.md:24]: @Cronus1007 I believe branching arrows and brackets over multiple steps would be a bit jarring and possibly confusing to see in markdown.

Comment (Cronus1007) [reply CONTRIBUTING.md:24]: I think you took the wrong path. I am telling to code in the markdows and while seeing the documentation we can add the Previewed model of the markdown.

**Comment by Cronus1007** [comment]
@brylie Have a look at this PR. Since issue #304 looks pretty important to me.

**Comment by brylie** [comment]
I agree that issue #304 would be valuable and suggest that we close this pull request in favor of #304.

**Pull request closed without merging**